src/Controller/Api/PersonController.php

```diff
@@ -450,11 +451,11 @@ public function deleteAction(int $id)
         $primaryPointOfContactCount = $this->entityHandler->count(DIF::class, array('primaryPointOfContact' => $id));
         $secondaryPointOfContactCount = $this->entityHandler->count(DIF::class, array('secondaryPointOfContact' => $id));
         if ($primaryPointOfContactCount > 0) {
-            throw new BadRequestHttpException('This Person is not deletable because 
+            throw new BadRequestHttpException('This Person is not deletable because
```

Comment:
I think that space was needed here. We shouldn't do multi-line strings really.

Reply:
ended up making string outside of throw...too long?